src/pydata_sphinx_theme/assets/scripts/bootstrap.js

```diff
@@ -1,18 +1,28 @@
-/* Import and setup functions to control Bootstrap's behavior.
- * Sphinx injects the html output with jquery and other javascript files. To enable
- * Popper.js (and other jQuery plugins) to hook into the same instance of jQuery,
- * jQuery is defined as a Webpack external, thus this import uses the externally defined jquery dependency.
- */
-import "jquery";
-import "popper.js";
-import "bootstrap";
+// Import and setup functions to control Bootstrap's behavior.
+import "@popperjs/core";
+import { Tooltip } from "bootstrap";
+import { documentReady } from "./mixin";
 
 import "../styles/bootstrap.scss";
 
+/*******************************************************************************
+ * Trigger tooltips
+ */
+
+/**
+ * Add tooltip to each element with the "tooltip" data-bs-toogle class
+ */
+function TriggerTooltip() {
+  var tooltipTriggerList = [].slice.call(
+    document.querySelectorAll('[data-bs-toggle="tooltip"]')
+  );
+  tooltipTriggerList.map(function (tooltipTriggerEl) {
+    return new Tooltip(tooltipTriggerEl);
+  });
+}
+
 /*******************************************************************************
  * Call functions after document loading.
- * This is equivalent to the .ready() function as described in
- * https://api.jquery.com/ready/
  */
 
-$('[data-toggle="tooltip"]').tooltip({ delay: { show: 500, hide: 100 } });
+documentReady(TriggerTooltip);
```

Comment:
these now show up to the right of the icons instead of below. They also appear / fade in quicker than before.  Not broken, necessarily, but different.  I liked the tooltip below better than to the right, but I don't care about the fade in timing very much.